crates/subspace-proof-of-time/src/pot.rs

```diff
@@ -0,0 +1,101 @@
+//! Common defines.
+
+use sp_core::{Blake2Hasher, Hasher, H256};
+use std::ops::BitXor;
+
+/// The 128 bit key for the AES encryption.
+#[derive(Clone, Debug)]
+pub struct AesKey(pub [u8; 16]);
```

Comment:
Provide constructors, but make internal contents unavailable directly. Also this type should likely be moved into `subspace-core-primitives` and maybe not even called AES since AES is an implementation detail.

For example in `subspace-core-primitives` we have `PosSeed`. It is not saying that it is Chia, its implementation is irrelevant for most purposes (in fact I have added another proof of time implementation later for testing purposes and everything continued to work and make senses).

Reply:
Done